README.md

```diff
@@ -59,6 +74,27 @@ bedrock.start();
 
 For documentation on server configuration, see [config.js](./lib/config.js).
 
+### IP Tunnel
+Bedrock server maybe used with an IP tunnel.
+
+In `config/express.js`:
+```js
+// allow the tunnel http only access
+config.express.httpOnly = true;
+// configure fastify to trust the ip tunnel proxy 
+config.express.fastifyOptions.trustProxy = true;
+```
+
+In `config/server.js`:
+```js
+// accept connections from any IP
```

Comment:
Binding a service to 0.0.0.0 means that your service will listen on all it's network interfaces.

https://stackoverflow.com/questions/71681234/what-does-bind-0-0-0-0port-mean-in-node-command-line#:~:text=Binding%20a%20service%20to%200.0,inside%20your%20local%20network%20(192.168.

Reply:
`0.0.0.0` means a lot of different things I've found so thanks for the clarification. Made the change here: https://github.com/digitalbazaar/bedrock-server/pull/33/commits/f9694dac1f9ca477c3f83f26c5c8192953ff2639